halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/openstack/OpenstackEditAccountCommand.java

```diff
@@ -75,8 +87,32 @@ protected String getProviderName() {
   private Boolean insecure;
 
   @Parameter(
-      names = "--user-data-file",
+      names = "--heat-template-location",
       converter = PathExpandingConverter.class,
+      description = OpenstackCommandProperties.HEAT_TEMPLATE_LOCATION_DESCRIPTION
+  )
+  private String heatTemplateLocation;
+
+  @Parameter(
+      names = "--remove-heat-template-location",
+      description = "Removes currently configured heat template location."
+  )
+  private boolean removeHeatTemplateLocation;
+
+  @Parameter(
+      names = "--consul-config",
+      description = OpenstackCommandProperties.CONSUL_CONFIG_DESCRIPTION
+  )
+  private String consulConfig;
+
+  @Parameter(
+      names = "--remove-consul-config",
+      description = "Removes currently configured consul config file."
+  )
+  private boolean removeConsulConfig;
```

Comment:
These "-remove" style flags aren't needed. You can always clear with `--consul-config=`.